src/state/opamSysInteract.ml

```diff
@@ -286,30 +286,9 @@ let packages_status packages =
     in
     compute_sets sys_installed ~sys_available
   | Centos ->
-    (* XXX /!\ only checked on centos XXX *)
-    let lines = run_query_command "yum" ["-q"; "-C"; "list"] in
-    (* -C to retrieve from cache, no update but still quite long, 1,5 sec *)
-    (* Return a list of installed packages then available ones:
-       >Installed Packages
-       >foo.arch    version   repo
-       >Available Packages
-       >bar.arch    version   repo
-    *)
-    let sys_installed, sys_available, _ =
-      List.fold_left (fun (inst,avail,part) -> function
-          (* beware of locales!! *)
-          | "Installed Packages" -> inst, avail, `installed
-          | "Available Packages" -> inst, avail, `available
-          | l ->
-            (match part, OpamStd.String.split l '.' with
-             | `installed, pkg::_ ->
-               pkg +++ inst, avail, part
-             | `available, pkg::_ ->
-               inst, pkg +++ avail, part
-             | _ -> (* shouldn't happen *) inst, avail, part))
-        OpamSysPkg.Set.(empty, empty, `preamble) lines
-    in
-    compute_sets sys_installed ~sys_available
+    let lines = run_query_command "rpm" ["-qa"; "--qf"; "%{NAME}\\n"] in
```

Comment:
Could you provide an output format example, is the same form than the other ones in the file (`>` are used as bol, to highlight spaces)